src/renderer/tvgLoadModule.h

```diff
@@ -64,6 +65,7 @@ struct LoadModule
     virtual bool open(const char* data, uint32_t size, const char* rpath, bool copy) { return false; }
     virtual bool resize(Paint* paint, float w, float h) { return false; }
     virtual void sync() {};  //finish immediately if any async update jobs.
+    virtual bool resolve(const AssetResolver* resolver) { return false; }
```

Comment:
This is lottie specific feature, shouldn't be general.
See: `struct LottieLoader`